plugins/manipulator.cpp

```diff
@@ -807,7 +1735,9 @@ class viewscreen_unitbatchopst : public dfhack_viewscreen {
         formatter.add_option("i", "Position in list", unit_ops::get_list_id);
         formatter.add_option("pi", "Position in list, among dwarves with same profession", unit_ops::get_list_id_prof);
         formatter.add_option("gi", "Position in list, among dwarves in same profession group", unit_ops::get_list_id_group);
-        formatter.add_option("ri", "Raw unit ID", unit_ops::get_unit_id);
```

Comment:
Why was this one removed? I think the other two are ones you added for debugging purposes, or something like that.

Reply:
I mistook it for one of my debug fields.